app/views/membership_applications/_reason_waiting.html.haml

```diff
@@ -28,13 +28,38 @@
                               method: 'put' } }
 
 
-
 :javascript
 
-  $('#member_app_waiting_reasons').on('ajax:success', function (e, data) {
-    if (data === "#{@other_waiting_reason_value}") {
-      $('#other-text-field').show();
-    } else {
-      $('#other-text-field').hide();
-    }
+  var ChangeStatus = {
```

Comment:
We're still on poltergeist for all javascript cucumber tests.  When we've expanded to include selenium/chrome as an option we can use the latest JS version.